persistence/sql/persister_identity.go

```diff
@@ -377,26 +377,38 @@ func (p *Persister) HydrateIdentityAssociations(ctx context.Context, i *identity
 	return p.injectTraitsSchemaURL(ctx, i)
 }
 
-func (p *Persister) ListIdentities(ctx context.Context, expand identity.Expandables, page, perPage int) (res []identity.Identity, err error) {
+func (p *Persister) ListIdentities(ctx context.Context, params identity.ListIdentityParameters) (res []identity.Identity, err error) {
 	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.ListIdentities")
 	defer otelx.End(span, &err)
 	span.SetAttributes(
-		attribute.Int("page", page),
-		attribute.Int("per_page", perPage),
-		attribute.StringSlice("expand", expand.ToEager()),
+		attribute.Int("page", params.Page),
+		attribute.Int("per_page", params.PerPage),
+		attribute.StringSlice("expand", params.Expand.ToEager()),
+		attribute.String("credential_identifier", params.CredentialsIdentifier),
```

Comment:
Could this be considered PII? I guess not, since it's without context here. What's the policy on email addresses in traces?

Reply:
Very good point, we should maybe just say whether a filter was included or not

Reply:
done